Dockerfile

```diff
@@ -1,27 +1,23 @@
-FROM quay.io/centos/centos:7
+FROM quay.io/rockylinux/rockylinux:9
 
 # elixir expects utf8.
 ENV LANG=en_US.utf8
 
 ENV ERLANG_VERSION=24.2.1
+ENV ERLANG_DOWNLOAD_URL="https://github.com/rabbitmq/erlang-rpm/releases/download/v${ERLANG_VERSION}/erlang-${ERLANG_VERSION}-1.el9.x86_64.rpm"
+ENV ERLANG_DOWNLOAD_SHA512="297fd37bf1d2518f4a9ec38534d03332b0e804131aa5dd2ee69d69cbb4dacff3ebc279e8305eb4be53843e9c113276fc4574ba8505b07f406b98f546aefb176e"
 
 ENV ELIXIR_VERSION="v1.13.2"
 ENV ELIXIR_DOWNLOAD_URL="https://github.com/elixir-lang/elixir/archive/${ELIXIR_VERSION}.tar.gz"
 ENV ELIXIR_DOWNLOAD_SHA512="62fe42a9c3439d1fcba31ce4f9b0e489e6ad71417dc7de200b1651cd61a1fc5c433b1b9e0ecc293a5e5c279da47c909df45253193f6b4a8253043e07def05bc4"
 
-ENV NODESOURCE="https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm"
-
 RUN set -xe \
 	&& yum clean all && yum update -y \
-	&& yum install -y yum-plugin-copr \
-	&& yum copr enable -y mbrancaleoni/erlang \
-	&& yum install -y ${NODESOURCE} \
-	&& yum install -y rpm-build createrepo epel-release make git lsof openssh-clients which nodejs sox openssl mariadb yajl \
-	&& yum groups mark install "Development Tools" \
-	&& yum groups mark convert "Development Tools" \
-	&& yum groupinstall -y "Development Tools" \
```

Comment:
The "Development tools" group is needed to build C extensions. There're many erlang/elixir libs that build some C module and w/o development tools they cannot be built.